remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerInterceptor.java

```diff
@@ -278,6 +278,10 @@ private long copyFlags(FlagAffectedCommand command) {
 
    @Override
    public Object visitPutMapCommand(InvocationContext ctx, PutMapCommand command) {
+      if (!ctx.isOriginLocal()) {
```

Comment:
Is this a different bug?

Reply:
Yes: https://github.com/infinispan/infinispan/pull/7003/commits/a8937b8f778f3d1c05f6e779778f56e0dd6384b9

Reply:
But should we merge that here? Or do you want another PR as you mentioned in previous comment?